coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/projectadministration/project/repository/ProjectRepository.java

```diff
@@ -159,6 +159,26 @@
           + "CREATE (p)-[r:CONTAINS_COMMIT]->(c)")
   void attachCommitsWithIds(long projectId, @NonNull List<Long> commitIds);
 
+  /** @param projectId The id of the project */
   @Query("MATCH (p)<-[r:WORKS_ON]-() WHERE ID(p) = {0} DELETE r")
   void deleteContributorRelationships(long projectId);
+
+  /**
+   * @param userId The user id.
+   * @return All the project a user is assigned to.
+   */
+  @Query(
+      "MATCH (u:UserEntity) WHERE ID(u) = {0} WITH u "
+          + "OPTIONAL MATCH (u)-[:ASSIGNED_TO*0..1]->(p1:ProjectEntity) WHERE p1.isBeingDeleted = FALSE "
+          + "WITH p1, u "
+          + "MATCH (p2:ProjectEntity)<-[:ASSIGNED_TO*0..1]-(t)<-[:IS_IN*0..1]-(u)  WHERE p2.isBeingDeleted = FALSE WITH collect(p1) + collect(p2) as list "
+          + "UNWIND list AS p RETURN DISTINCT p ORDER BY p.name")
+  List<ProjectEntity> findProjectsByUsedId(long userId);
+
+  /**
+   * @param teamId The team id.
+   * @return All the project a team is assigned to.
+   */
+  @Query("MATCH (t)-[:ASSIGNED_TO]->(p) WHERE ID(t) = {0} AND p.isBeingDeleted = FALSE RETURN p")
+  List<ProjectEntity> listProjectByTeamId(long teamId);
```

Comment:
```suggestion
  List<ProjectEntity> listProjectsByTeamId(long teamId);
```
Entsprechend müssten alle Verwendungen angepasst werden.

Reply:
Fixed